frontend/src/App.tsx

```diff
@@ -107,6 +109,13 @@ const App = (): React.ReactElement => {
                   roles={["Admin", "Teacher"]}
                 />
                 <Route exact path="*" component={NotFound} />
+                <Route
+                  exact
+                  path={Routes.NAME_SELECTION}
+                  component={NameSelection}
+                />
```

Comment:
this has to go before the line `<Route exact path="*" component={NotFound} />` since this is a switch statement